File: 3232-FindIfDigitGameCanBeWon/javascript/solution.js
// Last updated: 3/6/2026, 12:39:18 AM
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var canAliceWin = function(nums) {
    let single = 0;
    let double = 0;
    for(let num of nums){
        if(num>=10){
            double += num;
        }
        else{
            single += num;
        }
    }
    if(single>double||single<double){
        return true;
    }
    else{
        return false;
    }
};

File: 3232-FindIfDigitGameCanBeWon/cpp/solution.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int single = 0;
        int Double = 0;
        for (int num : nums) {
            if (num >= 10) {
                Double += num;
            } else {
                single += num;
            }
        }
        if (single > Double || single < Double) {
            return true;
        } else {
            return false;
        }
    }
};